README.rst

```diff
@@ -64,6 +64,16 @@ Zgen_
 
     zgen load "MichaelAquilina/zsh-auto-notify"
 
+Fig_
+
+Fig adds apps, shortcuts, and autocomplete to your existing terminal.
```

Comment:
I would prefer we dont have this here actually - I understand that it's explaining what `Fig` is, but I think the installation steps should just explain how to use install this.

If a user already uses Fig, then they would know what it is already it is.

This is the same approach taken for all other managers in the README

Reply:
Understandable, removed that sentence from all 3 PRs.